src/main/java/de/tum/in/www1/artemis/web/rest/open/PublicPyrisStatusUpdateResource.java

```diff
@@ -65,4 +66,31 @@ public ResponseEntity<Void> setStatusOfJob(@PathVariable String runId, @RequestB
 
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * {@code POST /api/public/pyris/pipelines/course-chat/runs/{runId}/status} : Set the status of a course chat job.
```

Comment:
```suggestion
     * POST public/pyris/pipelines/course-chat/runs/{runId}/status : Set the status of a course chat job.
```
This is inconsistent with the formatting in other Resources (even within this PR).